stable/cluster-autoscaler/values.yaml

```diff
@@ -53,7 +53,7 @@ cloudConfigPath: /etc/gce.conf
 
 image:
   repository: k8s.gcr.io/cluster-autoscaler
-  tag: v1.14.6
+  tag: v1.17.1
```

Comment:
can you update readme too for consistency? (also fine with having a value like chart.appversion in the docs

Reply:
Sure, will do.

Reply:
Pushed in this commit: https://github.com/helm/charts/pull/21311/commits/5250e2f7755d5ddeb6640ae0796c340e04f0568e

Reply:
It seems likely `values.yaml` and `README.md` will fall out of sync at some point, and thus we spread misinformation. What's the benefit of repeating in the README?